tests/com.espressif.idf.ui.test/src/com/espressif/idf/ui/test/executable/cases/project/NewEspressifIDFProjectPartitionTableEditorTest.java

```diff
@@ -31,213 +32,160 @@
 @SuppressWarnings("restriction")
 @RunWith(SWTBotJunit4ClassRunner.class)
 @FixMethodOrder(MethodSorters.NAME_ASCENDING)
-public class NewEspressifIDFProjectPartitionTableEditorTest
-{
+public class NewEspressifIDFProjectPartitionTableEditorTest {
+
 	@BeforeClass
-	public static void beforeTestClass() throws Exception
-	{
+	public static void beforeTestClass() throws Exception {
 		Fixture.loadEnv();
+		Fixture.createAndBuildProject("Project1");
+		Fixture.createProject("Project2"); // not built
```

Comment:
Use more descriptive names so comments aren’t needed. For example, you can name the first one "BuiltProject" and the second "CleanProject". Also, use constants to store these names since they are used multiple times in this class.